doc/policy/packages.md

```diff
@@ -57,3 +57,40 @@ test accepts):
 
    - Warning: Batched fee-bumping may be unsafe for some use cases. Users and application developers
      should take caution if utilizing multi-parent packages.
+
+### Package Fees and Feerate
+
+*Package Feerate* is the total modified fees divided by the total virtual size of all transactions
+in the package after deduplication.
```

Comment:
It's unclear to me what "deduplication" means here. I thought a package, by definition, contains no duplicate transactions.